VAMobile/src/components/FormWrapper/FormWrapper.test.tsx

```diff
@@ -105,7 +104,7 @@ context('FormWrapper', () => {
         initializeTestInstance([shortenedFieldsList])
         testInstance.findByType(VATextInput).props.setError('')
         const textViews = testInstance.findAllByType(TextView)
-        expect(textViews[textViews.length - 2].props.children).not.toEqual('first error message')
+        expect(textViews[textViews.length - 1].props.children).not.toEqual('first error message')
```

Comment:
Change this to the new way unit tests are done. If you need an example look at my SM analytics ticket.

use expect(screen.getByText('your text').toBeTruthy() if it is supposed to be there and queryByText('your text).toBeFalsy() if it is not.

Reply:
Good point, since I was already here I went ahead and updated the other tests in the file as well